unity/Holo/Assets/Editor/ModelImport/FileSeriesImporter.cs

```diff
@@ -0,0 +1,174 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+
+namespace ModelImport
+{    
+    //A class for importing a file series from a directory.
+    public class FileSeriesImporter
+    {
+        public GameObject ModelGameObject { get; private set; }
+        public Mesh ModelMesh { get; private set; }
+
+        public GameObject[] gameObjectArray;
+        public Mesh[] meshArray;
+        private string[] filePaths;
+        private string extension;
+        private FileImporter fileImporter;
+        private Dictionary<string, Vector3> boundingVertices = new Dictionary<string, Vector3>();
+        private bool dataflow = false;
+        
+        // Object constructor, initiates file series import.
+        public void ImportData(string fileSeriesDirectory, string gameObjectName)
+        {
+            ModelGameObject = new GameObject
+            {
+                name = gameObjectName
+            };
+            ModelMesh = new Mesh();
+            if (gameObjectName.Contains("dataflow"))

```

Comment:
Is there any other way to determine `dataflow` indication?

Reply:
As of now, not really. We could leave some indication in the vtk files we do, but for now that's the only flag that needs to be passed down.